types/Psychologist.ts

```diff
@@ -0,0 +1,25 @@
+export type Psychologist = {
+    dossierNumber: string,
+    firstNames: string,
+    lastName: string,
+    archived: boolean,
+    state: string,
+    adeli: string,
+    address: string,
+    diploma: string,
+    phone: string,
+    email: string,
+    personalEmail: string,
+    website: string,
+    teleconsultation: boolean,
+    description: string,
+    training: string,
+    departement: string,
+    university: string,
+    declaredUniversityId?: string,
+    assignedUniversityId: string,
+    region: string,
+    languages: string,
+    active: boolean,
+    inactiveUntil: string | undefined,
```

Comment:
Shouldn't `inactiveUntil` be a `Date`?

Reply:
Again for my personal knowledge (learn JS with Xavier by reviewing him :p): what's the difference between `inactiveUntil?: string` and `inactiveUntil: string | undefined`?

Reply:
not clear now if it's a string or a date, I think it can be both depending on if you are coming from the db or from the front
for now i'd rather have it based on usage until we do a proper ts migration

Reply:
That's a very good question
actually the string | undefined does not make sens because a field can always be undefined
the difference is that if you use `incativeUntil: string | undefined` you have to specify it
so this is ok `{active: true, inactiveUntil: undefined}` but this is not `{active: true}`
if you put it optional `inactive?: string`, then you can do `{active: true}`